client/src/components/Footer.tsx

```diff
@@ -1,167 +1,103 @@
-import { useEffect, useState } from "react";
-import { useTheme } from "next-themes";
-import { DiscordIcon } from "../images/Discord";
-import { GithubIcon } from "../images/Github";
-import { TwitterIcon } from "../images/Twitter";
-import { styled } from "../stitches.config";
-import { Box } from "../ui/Box";
-import { Container } from "../ui/Container";
-import { Link } from "../ui/Link";
-import { Text } from "../ui/Text";
-
-const SocialIconLink = styled("a", {
-  mx: "$2",
-  p: "$2",
-  br: "$1",
-  color: "$gray1",
-  "&:hover": {
-    backgroundColor: "$gray10",
-  },
-});
+import { useEffect, useState } from 'react';
+import { useTheme } from 'next-themes';
+import { Box } from '../ui/Box';
+import { Button } from '../ui/MenuButton';
+import { IconButton } from '../ui/IconButton';
+import { SunIcon } from '@radix-ui/react-icons';
 
 export const Footer = () => {
-  const { theme } = useTheme();
+  // setting theme in from the footer in sidebar
+  const { theme, setTheme } = useTheme();
   const [mounted, setMounted] = useState(false);
 
   useEffect(() => setMounted(true), []);
 
   return (
-    <Box
-      css={{ py: "$8", mt: 64, backgroundColor: "$gray12", color: "$gray1" }}
-    >
-      <Container>
+    <>
+      <Box as={'li'} css={{ py: '$1', px: '$1' }}>
+        <Box
+          as={'a'}
+          css={{
+            textDecoration: 'none',
+            color: '$gray12',
+            '&.active': {
+              color: '$primary',
+              fontWeight: 700,
+            },
+          }}
+          href="https://twitter.com/sigleapp"
+          target="_blank"
+        >
+          <Button
+            css={{
+              borderRadius: '$1',
+            }}
+          >
+            Twitter
+          </Button>
+        </Box>
+      </Box>
+
+      <Box as={'li'} css={{ py: '$1', px: '$1' }}>
         <Box
+          as={'a'}
           css={{
-            display: "flex",
-            alignItems: "center",
-            justifyContent: "space-between",
-            flexDirection: "column",
-            "@lg": {
-              flexDirection: "row",
+            textDecoration: 'none',
+            color: '$gray12',
+            '&.active': {
+              color: '$primary',
+              fontWeight: 700,
             },
           }}
+          href="https://discord.gg/X2Dbz3xbrs"
+          target="_blank"
         >
-          <Box
+          <Button
```

Comment:
I think this could be simplified. The Button component could be merged with its parent (Box) to simplify a bit this part wdyt?